### PR TITLE
Update typegen to handle numeric(x,y) types

### PR DESCRIPTION
**Pull request by johncsnyder**
Fixes https://github.com/mmkal/slonik-tools/issues/415

### CODE REVIEW

File: packages/typegen/test/options.test.ts [hunk 1]
```diff
@@ -30,7 +30,9 @@ beforeEach(async () => {
       j json,
       jb jsonb,
       j_nn json not null,
-      jb_nn jsonb not null
+      jb_nn jsonb not null,
+      d_p numeric(8),
+      d_ps numeric(8,4)
```
Comment (mmkal) [inline packages/typegen/test/options.test.ts:35]: How about 
```suggestion
      d_ps numeric(8, 4)
```
?

File: packages/typegen/src/index.ts [hunk 2]
```diff
@@ -114,7 +114,7 @@ export const generate = async (params: Partial<Options>) => {
       return itemType.match(/^\w+$/) ? `${itemType}[]` : `Array<${itemType}>`
     }
 
-    if (regtype.match(/\(\d+\)/)) {
+    if (regtype.match(/\([\d,]+\)/)) {
```
Comment (mmkal) [inline packages/typegen/src/index.ts:117]: Should probably allow for whitespace too

File: packages/typegen/test/options.test.ts (hunk 1 above)
Comment (johncsnyder) [reply packages/typegen/test/options.test.ts:35]: As far as I know, postgres strips the whitespace from the type. But I don't see the harm in covering the whitespace case too.

File: packages/typegen/src/index.ts (hunk 2 above)
Comment (johncsnyder) [reply packages/typegen/src/index.ts]: Ok done :)

**Pull request merged**